mod/node-core/pkg/config/flags/flags.go

```diff
@@ -26,12 +26,15 @@ const (
 	BeaconKitAcceptTos = beaconKitRoot + "accept-tos"
 
 	// Builder Config.
-	builderRoot              = beaconKitRoot + "builder."
+	builderRoot              = beaconKitRoot + "payload-builder."
 	SuggestedFeeRecipient    = builderRoot + "suggested-fee-recipient"
-	Graffiti                 = builderRoot + "graffiti"
 	LocalBuilderEnabled      = builderRoot + "local-builder-enabled"
 	LocalBuildPayloadTimeout = builderRoot + "local-build-payload-timeout"
 
+	// Validator Config.
```

Comment:
@ocnc2 is the template.go okay?

Reply:
yep template.go already has this config reflected